solidity/contracts/bridge/Bridge.sol

```diff
@@ -138,12 +197,69 @@ contract Bridge is Ownable {
     ///         execute a sweep.
     mapping(uint256 => DepositInfo) public deposits;
 
-    /// @notice Maps the wallet public key hash (computed using HASH160 opcode)
-    ///         to the latest wallet's main UTXO computed as
+    /// @notice Maps the 20-byte wallet public key hash (computed using HASH160
+    ///         opcode) to the latest wallet's main UTXO computed as
```

Comment:
We could explain that the HASH160 is done on 33-byte compressed ECDSA public key. (Assuming this is true.)

Reply:
See dbb7bf37